docs/source/get_started/new_project.md

```diff
@@ -28,29 +27,91 @@ So, if you enter "Get Started", the folder for the project (`repo_name`) is auto
 
 ### Project tools
 
-The command line interface then asks which tools you'd like to include in the project. The options are as follows and described in more detail above in the [documentation about the new project tools](../starters/new_project_tools.md).
+Next, the CLI asks which tools you'd like to include in the project:
 
-You can add one or more of the options, or follow the default and add none at all:
+```text
+Which tools would you like to include in your project? [1-7/1,3/all/none]:
+
+Tools
+1) Lint: Basic linting with Black and Ruff
+2) Test: Basic testing with pytest
+3) Log: Additional, environment-specific logging options
+4) Docs: A Sphinx documentation setup
+5) Data Folder: A folder structure for data management
+6) PySpark: Configuration for working with PySpark
+7) Kedro-Viz: Kedro's native visualisation tool
+```
+
+The options are described in more detail in the [documentation about the new project tools](../starters/new_project_tools.md).
+
+Select the tools by number, or `all` or follow the default to add `none`.
 
-* Linting: A basic linting setup with Black and ruff
-* Testing: A basic testing setup with pytest
-* Custom Logging: Additional logging options
-* Documentation: Configuration for basic documentation built with Sphinx
-* Data Structure: The [directory structure](../faq/faq.md#what-is-data-engineering-convention) for storing data locally
-* PySpark: Setup and configuration for working with PySpark
-* Kedro Viz: Kedro's native visualisation tool.
 
 ### Project examples
 
-The CLI offers the option to include example pipelines. Your choice of tools determines which spaceflights starter example is provided. Here's a guide to understanding which starter examples are used based on your selections:
+Finally, the CLI offers the option to include starter example code in the project:
 
-* [Default Starter](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas) (`spaceflights-pandas`): Used when you select any combination of Linting, Testing, Custom Logging, Documentation, and Data Structure, excluding PySpark and Kedro Viz.
-* [PySpark Starter](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pyspark) (`spaceflights-pyspark`): Chosen when PySpark is selected with any other tools, except Kedro Viz.
-* [Kedro Viz Starter](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas-viz) (`spaceflights-pandas-viz`): Applicable when Kedro Viz is part of your selection, with any other tools, excluding PySpark.
-* [Full Feature Starter](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pyspark-viz) (`spaceflights-pyspark-viz`): This example is used when you select all available tools, including PySpark and Kedro Viz.
+```text
+Select whether you would like an example spaceflights pipeline included in your project.
+```
```

Comment:
This is what I got from `main`:

````suggestion
```text
Would you like to include an example pipeline? :
 (no): 
```
````

Again, excluding some intro lines:

```
Example Pipeline
================
Select whether you would like an example spaceflights pipeline included in your project.
To skip this step in the future use --example=y/n
To find out more: https://docs.kedro.org/en/stable/starters/new_project_tools.html
```